### PR TITLE
chore: replacing biohub logo in footer

### PR DESCRIPTION
**Pull request by sethfeingold**
simple update to biohub logo in footer

### CODE REVIEW

File: src/frontend/src/views/LandingPageV2/components/Footer/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
 import { useEffect } from "react";
-import BiohubLogoImg from "src/common/images/cz-biohub-logo.png";
+import BiohubLogoImg from "src/common/images/biohub-sf-logo.svg";
```
Comment (mmmmmmaya) [inline src/frontend/src/views/LandingPageV2/components/Footer/index.tsx]: It seems like this is the only use of the old logo in the repo. What do you think about removing it?

File: src/frontend/src/views/LandingPageV2/components/Footer/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
 import { useEffect } from "react";
-import BiohubLogoImg from "src/common/images/cz-biohub-logo.png";
```
Comment (mmmmmmaya) [inline src/frontend/src/views/LandingPageV2/components/Footer/index.tsx:3]: It seems like you replaced a png with an svg, which actually does not work as a direct swap out. With this code, the logo is not visible.
Screenshot 2023-03-28 at 9 46 37 AM

File: src/frontend/src/views/LandingPageV2/components/Footer/index.tsx (hunk 1 above)
Comment (sethfeingold) [reply src/frontend/src/views/LandingPageV2/components/Footer/index.tsx]: Definitely okay with removing the asset from the images dir 👍🏻

File: src/frontend/src/views/LandingPageV2/components/Footer/index.tsx (hunk 2 above)
Comment (sethfeingold) [reply src/frontend/src/views/LandingPageV2/components/Footer/index.tsx:3]: Replaced with png version 😄  thanks for the heads up!

**Review by mmmmmmaya — APPROVED** [review]

**Comment by sethfeingold** [comment]
Thanks so much for the review and help @mmmmmmaya 😄

**Pull request merged**